jwst/master_background/master_background_step.py

```diff
@@ -0,0 +1,86 @@
+from ..stpipe import Step
+from .. import datamodels
+
+
+__all__ = ["MasterBackgroundStep"]
+
+
+class MasterBackgroundStep(Step):
+    """
+    MasterBackgroundStep:  Compute and subtract master background from spectra
+    """
+
+    spec = """
+        user_background = str(default=None) # Path to user-supplied master background
+        save_background = boolean(default=False) # Save computed master background
+    """
+
+
+    def process(self, input):
+
+        """
+        Compute a master background from N > 1 dedicated background exposures
```

Comment:
I would leave this a bit more generic, such as "Compute a master background spectrum", because it's not always the case that the MSB will come from dedicated background exposures. For NIRSpec MSA, for example, it'll come from background slitlets in the same exposure.